src/field.cpp

```diff
@@ -323,6 +323,13 @@ void game::init_fields()
             {"", "", ""}, '&', -1,
             {c_white, c_yellow, c_red}, {true, true, true}, {false, false, false}, MINUTES(50),
             {0,0,0}
+        },
+
+        {
+            "fd_fungicidal_gas",
+            {_("hazy cloud"),_("fungicidal gas"),_("thick fungicidal gas")}, '8', 8,
+            {c_white, c_ltgray, c_dkgray}, {true, true, false}, {false, true, true}, MINUTES(30),
```

Comment:
You could also alter the time here. The `MINUTES(30)` is the average time it takes for the field to decay by 1 density.